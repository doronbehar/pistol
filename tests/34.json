{
	"text": "A JSON file should be printed with JSON chroma syntax highlighting"
}
