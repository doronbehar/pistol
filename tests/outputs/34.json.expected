[38;2;248;248;242m{[0m[38;2;249;38;114m␊[0m
[38;2;102;217;239m·[0m[38;2;102;217;239m·[0m[38;2;248;248;242m"text":"A[0m[38;2;102;217;239m·[0m[38;2;248;248;242mJSON[0m[38;2;102;217;239m·[0m[38;2;248;248;242mfile[0m[38;2;102;217;239m·[0m[38;2;248;248;242mshould[0m[38;2;102;217;239m·[0m[38;2;248;248;242mbe[0m[38;2;102;217;239m·[0m[38;2;248;248;242mprinted[0m[38;2;102;217;239m·[0m[38;2;248;248;242mwith[0m[38;2;102;217;239m·[0m[38;2;248;248;242mJSON[0m[38;2;102;217;239m·[0m[38;2;248;248;242mchroma[0m[38;2;102;217;239m·[0m[38;2;248;248;242msyntax[0m[38;2;102;217;239m·[0m[38;2;248;248;242mhighlighting"[0m[38;2;249;38;114m␊[0m
[38;2;248;248;242m}[0m